typescript/cli/src/config/ism.ts

```diff
@@ -108,7 +115,11 @@ export async function createIsmConfigMap({
   outPath: string;
   chainConfigPath: string;
 }) {
-  logBlue('Creating a new ISM config');
+  logBlue('Creating a new advanced ISM config');
+  logBoldRed('WARNING: USE AT YOUR RISK.');
+  logRed(
+    'Advanced ISM config means you need to be very comfortable with different ISM types and how they work together topologically. If you are not, please use the basic ISM config instead.',
```

Comment:
This is good but I suggest a slightly different wording:
`Advanced ISM configs require knowledge of different ISM types and how they work together topologically. If possible, use the basic ISM configs are recommended.`